tests/allure/src/main/java/org/mule/test/allure/AllureConstants.java

```diff
@@ -1061,10 +1063,35 @@ interface OperationPolicyStory {
     }
   }
 
+  interface PricingMetricsFeature {
+
+    String PRICING_METRICS = "Pricing Metrics";
+
+    interface NetworkUsageMonitoringStory {
+
+      String MONITOR_NET_BYTES_USED = "Monitor net bytes used";
+    }
+
+    interface FlowSummaryStory {
+
+      String ACTIVE_FLOWS_SUMMARY = "Active flows summary";
+      String DETECT_APIKIT_FLOWS_AS_TRIGGERS = "Detect APiKit flows as triggers";
+
+    }
+  }
+
+  /**
+   * @deprecated Use {@link PricingMetricsFeature} or {@link NetworkUsageMonitoringStory} instead.
+   */
+  @Deprecated
   interface MonitoringSideCar {
 
     String MONITOR_NET_BYTES_USED = "Monitor net bytes used";
 
+    /**
+     * @deprecated Use {@link PricingMetricsFeature} or {@link NetworkUsageMonitoringStory} instead.
+     */
+    @Deprecated
```

Comment:
@mule-pachanady I am deprecating these because the sidecar is not a feature nor a story, it just refers to how those are impemented.